adam_core/coordinates/cometary.py

```diff
@@ -262,7 +262,7 @@ def from_cartesian(cls, cartesian: CartesianCoordinates) -> "CometaryCoordinates
 
         coords_cometary = cartesian_to_cometary(
             cartesian.values,
-            cartesian.time.to_astropy().tdb.mjd,
+            cartesian.time.to_numpy(),
```

Comment:
I've decided to add to_numpy as a convenience method for the conversion to TDD-scaled, MJD-epoched, double-precisioned timestamps. Perhaps the name is opaque?

Reply:
Another option (now having read more of the code) might be to enable symmetry between `.from_mjd` by adding a `to_mjd(scale="tdb")`. The current `mjd()` could suffice as well with a scale parameter. The latter could just return a pyarrow array (the additional call to `.to_numpy()` from there doesn't seem super cumbersome from my perspective). 

I suspect a lot of astronomers will expect an interface similar to astropy. Don't know how much we want to mimic it but that's a thought. This doesn't have to be addressed in this PR obviously. Here are some options:

```python
import numpy as np
from adam_core.time import Timestamp

mjd_tdb = np.arange(59000, 60000)

times = Timestamp.from_mjd(mjd_tdb, scale="tdb") # this feels very nice and not much different than astropy
mjd_utc = times.utc().mjd().to_numpy() 
mjd_utc = times.mjd(scale="utc").to_numpy() 
mjd_utc = times.to_numpy(scale="utc")
mjd_utc = times.to_mjd(scale="utc") # the symmetry between .from_mjd() and .to_mjd() feels quite intuitive
```